pkg/test/acpi.go

```diff
@@ -19,6 +19,7 @@ var (
 		Required:                true,
 		function:                CheckRSDPValid,
 		Status:                  Implemented,
+		Firmware:                FWNoCoreboot,
```

Comment:
The new Field Firmware is just wrong, whatever it does. That's not clear from the commit message.